src/test/java/net/sf/log4jdbc/PreparedStatementSpyTest.java

```diff
@@ -31,52 +41,39 @@ void resetLoggers() {
     static class LoggingParamsIsDisabledCase implements Case {
 
         @Override
-        public void exec() {
-            String sql = "INSERT INTO People VALUES (?, ?);";
-            ConnectionSpy cs = mock(ConnectionSpy.class);
-            PreparedStatement ps = mock(PreparedStatement.class);
-
-            PreparedStatementSpy pss = new PreparedStatementSpy(sql, cs, ps);
-            assertEquals(sql, pss.dumpedSql());
+        public void exec() throws SQLException {
+            try (final var pss = prepareInsertIntoPeopleStmtSpy(2, "Ronaldo")) {
```

Comment:
Nice touch with autocloseable, but only mocks are passed to the constructor of `PreparedStatementSpy`, so I ignored it completely because it neither consumes any resources, nor test case depends on flushing of the stuff in flight.